packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -217,6 +218,13 @@ class Draggable<T extends Object> extends StatefulWidget {
   /// The data that will be dropped by this draggable.
   final T? data;
 
+  /// The mouse cursor for mouse pointers that are over the [feedback].
+  ///
+  /// When a mouse is over the [feedback], its cursor will be changed to the [cursor].
```

Comment:
This seems to be redundant with the first sentence?

Reply:
> This seems to be redundant with the first sentence?

Rewrote, I hope it will be cleaner.